custom_components/tuya_local/goldair_101cm_tower_fan.yaml

```diff
@@ -0,0 +1,53 @@
+name: Tower Fan
+products:
+  - id: xonvukal96pf99s6
+    name: Goldair 101cm Tower Fan
+primary_entity:
+  entity: fan
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+    - id: 2
+      type: string
+      name: preset_mode
+      mapping:
+        - dps_val: 1
+          value: Normal
```

Comment:
These should be all lower case, and `translation_key: fan_with_presets` added to the entity definition.